processing/src/main/java/org/apache/druid/frame/key/RowKey.java

```diff
@@ -109,8 +109,12 @@ public String toString()
     return Arrays.toString(key);
   }
 
-  public int getNumberOfBytes()
+  /**
+   * Estimate number of bytes taken by an object of {@link RowKey}. Only returns an estimate and does not account for
+   * platform or JVM specific implementation.
+   */
+  public int estimatedObjectSize()
```

Comment:
I think we should also append a memory unit to the method name like `estimatedObjectSizeBytes`

Reply:
Done